tests/inputHandlerTest.js

```diff
@@ -0,0 +1,27 @@
+import InputHandler from '../src/lib/inputHandler';
+
+let input = new InputHandler(document.querySelector('html'));
```

Comment:
We typically don't wanna bind listeners to the html element, since it contains head and body, technically body is the space where the dom is rendered. 

```suggestion
let input = new InputHandler(document.querySelector('body'));
```

Reply:
Try to modify the inputhandler to take callbacks in it's options:

We expect the parameter "events" in the constructor to be a key value pair(JSON object) containing the following:
```
{
event_name: event_callback,
event_name: event_callback
}
```
In the constructor:
```
constructor(elem, events) {
...(earlier code here)

//add code to bind dynamic events to listeners
}
```

HINT use Object.Keys and look into Function.call()